tests/topotests/bgp_lu_multiple_labels/peer1/exa-send.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+"""
+exa-send.py: Send a few testroutes with ExaBGP
+"""
+
+from sys import stdout, argv
+from time import sleep
+
+sleep(5)
+
+stdout.write("announce route 2001:db8:100::/64 next-hop fc00::2 label [777 10006]\n")
```

Comment:
Can't we just announce these immediately without using a Python script (in the ExaBGP config directly)?